lib/battleships/grid.rb

```diff
@@ -1,27 +1,24 @@
-require 'battleships'
-require 'battleships/cell'
+require "battleships"
```

Comment:
This is just a matter of personal preference but please use single quotes here (and in similar places)